devicetypes/rachio/rachio-iro2-zone.src/rachio-iro2-zone.groovy

```diff
@@ -398,7 +277,7 @@ def scheduleDataEvent(data) {
     def elapsedDuration = data?.zoneStartDate ? getDurationMinDesc(Math.round(timeDiff)) : 0
     def wateringDuration = zoneDuration ? getDurationMinDesc(zoneDuration) : 0
     def zoneRunStatus = ((!zoneStartDate && !zoneDuration) || (zoneId != device.deviceNetworkId)) ?
-            "Status: Idle" : "${curSchedType == "automatic" ? "Scheduled" : "Manual"} Watering: ${elapsedDuration} of ${wateringDuration} Minutes"    
+            "Status: Idle" : "${curSchedType == "automatic" ? "Scheduled" : "Manual"} Watering: ${elapsedDuration} of ${wateringDuration} Minutes"
     def zoneCycleCount = (zoneId != device.deviceNetworkId && !data?.totalCycleCount) ? 0 : data?.totalCycleCount
 
```

Comment:
Why are we still sending events for attributes like "scheduleType" and "zoneDuration" when those  attributes have been deleted?